### PR TITLE
Added local build environment

### PR DESCRIPTION
**Pull request by KairaMiu**
This includes the following steps:
* Added golang docker builder image (KYT-150)
* Setup local build with dobi (KYT-151)

### CODE REVIEW

**Comment by KairaMiu** [comment]
Please check if name and place of docker image to be pushed is correct.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+      - id: flake8
+        args: [--max-line-length=130]
```
Comment (KlausPopp) [inline .pre-commit-config.yaml:13]: Add go pre-commit hooks (not yet tested!):

```
- repo: git://github.com/dnephin/pre-commit-golang
  rev: master
  hooks:
    - id: go-fmt
    - id: go-vet
    - id: go-lint
    - id: go-imports
    - id: go-cyclo
      args: [-over=15]
    - id: validate-toml
    - id: no-go-testing
    - id: gometalinter
    - id: golangci-lint
    - id: go-critic
    - id: go-unit-tests
    - id: go-build
    - id: go-mod-tidy
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Git pre-commit hook is used to ensure that a minimum set of quality is fullfilled by the checked in contents.
+
+The configured pre-commit hooks are:
+
+* Remove trailing whitespaces
+* Ensure files end in a newline and only a newline
+* Check yaml syntax
+* Prevent from committing a large files
+* Check python code style with flake8
```
Comment (KlausPopp) [inline README.md:21]: Add go checks here

**Review by KlausPopp — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (KairaMiu) [reply .pre-commit-config.yaml:13]: Great, I was looking for something like that but found nothing.

Comment (KairaMiu) [reply .pre-commit-config.yaml:13]: Added only pre-commit hooks for go, which require only golang to be pre-installed.

File: README.md (hunk 2 above)
Comment (KairaMiu) [reply README.md:21]: Added.

**Review by KlausPopp — APPROVED** [review]

**Pull request merged**